Platform/Samsung/sm8150/r5q.fdf.inc

```diff
@@ -0,0 +1,31 @@
+// per-device BSP DXEs
+
+FILE DRIVER = 8e9bd160-b184-11df-94e2-0800200c9a66 {
+    SECTION DXE_DEPEX = Platform/EFI_Binaries/Drivers/sm8150/DALSys/DALSys.depex
+    SECTION PE32 = Platform/EFI_Binaries/Drivers/Devices/vayu/DALSys/DALSys.efi
+    SECTION UI = "DALSys"
+}
+
+FILE DRIVER = 5bd181db-0487-4f1a-ae73-820e165611b3 {
+    SECTION DXE_DEPEX = Platform/EFI_Binaries/Drivers/sm8150/ButtonsDxe/ButtonsDxe.depex
+   SECTION PE32 = Platform/EFI_Binaries/Drivers/Devices/vayu/ButtonsDxe/ButtonsDxe.efi
```

Comment:
I don't understand why we should use ButtonsDxe and DALSys of other devices. Even though they can work, I don't think they should be used in this way